### ISSUE TITLE
Fix `declaration-property-value-no-unknown` false negatives for `text-wrap` and `text-wrap-mode`

### ISSUE THREAD

**Issue by Mouvedia** [stylelint/stylelint#7952]
### What minimal example or steps are needed to reproduce the bug?

```
a {
  text-wrap: foo;
  text-wrap-mode: foo;
}
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "rules": { "declaration-property-value-no-unknown": true }
}
```

### Resources

https://csstree.github.io/docs/validator.html
https://caniuse.com/mdn-css_properties_text-wrap
https://caniuse.com/mdn-css_properties_text-wrap-mode

**Comment by Mouvedia** [comment]
I've labeled the issue as ready to implement. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.

**Comment by sidverma32** [comment]
Have raised the fix [PR](https://github.com/stylelint/stylelint/pull/7956), Please have a look! @Mouvedia @ybiquitous

**Comment by romainmenke** [comment]
@Mouvedia I think it is better if we try to always follow the template and provide all the details for issues.

In this case it isn't clear from the description what the issue was or how it should be resolved.

I was trying to check if this was a false positive or false negative and couldn't tell without making a reproduction for myself.

-----

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdilAXGAHrgLQDuATkgA7woBmEEA3FjvkWZVcQLbQy0GzLAF8QAGhB0AlgBsYAOSQ84iQsqryAdGADOuieAiYZAcwQgM2FOhDkArvN23awGyFhhZSSrmnHiKnIIKhhyXABPYgA3JFl7GGJMCGJ7TABrZNJMFzwHGBQRUUNIE2lTADEIch4kXAsAK11jQ1gqA0QrHFtdSPlZaUxcXNtvfF7bcVZ3Xoj+wZJSs2JepEwoHygRkDGYCZBikSA)

```css
a {
  text-wrap: foo;
  text-wrap-mode: foo;
}
```

`foo` is not valid for either `text-wrap` or `text-wrap-mode` and an error is expected.

However no error was emitted with config:

```json
{
  "rules": { "declaration-property-value-no-unknown": true }
}
```

**Comment by Mouvedia** [comment]
@romainmenke My bad I was planning to fix it in #7957, but @sidverma32 beat me to it.
As you can see, the demo is listed in my PR's [OP](https://github.com/stylelint/stylelint/pull/7957#issue-2484635066).
Ill be more careful next time.
